convert2rhel/bootloader/bootloader.py

```diff
@@ -396,8 +283,8 @@ def _add_rhel_boot_entry(efibootinfo_orig):
 
     Return the new bootloader info (EFIBootInfo).
     """
-    dev_number = _get_device_number(get_efi_partition())
-    blk_dev = get_grub_device()
+    dev_number = _get_device_number(convert2rhel.bootloader.grub.get_efi_partition())
+    blk_dev = convert2rhel.bootloader.grub.get_grub_device()
```

Comment:
We could import grub and instead use this to make it simpler
```suggestion
    dev_number = _get_device_number(grub.get_efi_partition())
    blk_dev = grub.get_grub_device()
```